tests/system_tests/test_api_endpoints.py

```diff
@@ -0,0 +1,190 @@
+import time
+from typing import Any
+
+import pytest
+import requests
+from fastapi import status
+from pydantic import TypeAdapter
+
+from blueapi.client.rest import BlueapiRestClient, BlueskyRemoteControlError
+from blueapi.service.interface import get_devices, get_plans
+from blueapi.service.model import (
+    DeviceResponse,
+    EnvironmentResponse,
+    PlanResponse,
+    TasksListResponse,
+    WorkerTask,
+)
+from blueapi.worker.event import TaskStatusEnum, WorkerState
+from blueapi.worker.task import Task
+from blueapi.worker.task_worker import TrackableTask
+
+_SIMPLE_TASK = Task(name="sleep", params={"time": 0.0})
+_LONG_TASK = Task(name="sleep", params={"time": 1.0})
+
+rest = BlueapiRestClient()
```

Comment:
Should: Can you use `BlueapiClient` instead? Most of the calls here would just wrap `BlueapiRestClient` anyway but it makes for a good end-to-end test.